gulpfile.js

```diff
@@ -72,23 +68,35 @@ var source_html = [
 	'!./node_modules/**',
 	'!./dist/**'
 ];
-gulp.task('html', function() {
+gulp.task('html', function () {
 	gulp.src(source_html)
-	.pipe(debug({title: 'Move HTML:'}))
-	.pipe(gulp.dest('./dist'));
+		.pipe(debug({title: 'Move HTML:'}))
+		.pipe(gulp.dest('./dist'));
 });
 
 // Perform a complete build
-gulp.task('build', ['js', 'html', 'tests']);
+gulp.task('build', ['js', 'html', 'tests', 'sass']);
 
 
 // Watch and build on change
-gulp.task('watch', function() {  
+gulp.task('watch', function () {
 	gulp.watch(source_js, ['js']);
 	gulp.watch(source_test, ['tests']);
 	gulp.watch(source_html, ['html']);
+	gulp.watch(source_sass, ['sass']);
 	// gulp.watch(['./src/**/*.css', './src/**/*.css'], ['styles']);
 });
 
-
 gulp.task('default', ['watch']);
+
+var source_sass = [
+	'./pages/**/**.scss'
+];
+gulp.task('sass', function () {
+	gulp.src(source_sass)
+		.pipe(sass({outputStyle: 'expanded'}).on('error', sass.logError))
```

Comment:
here we have different options of the outputStyle that you can check and help deciding the way we produce & store our css files following [this](https://web-design-weekly.com/2014/06/15/different-sass-output-styles/) link.